src/main/fc/stats.c

```diff
@@ -56,29 +60,38 @@ void statsInit(void)
     dispatchEnable();
 }
 
+void writeStats(struct dispatchEntry_s* self);
+
+dispatchEntry_t writeStatsEntry =
+{
+    writeStats, 0, NULL, false
+};
+
 void writeStats(struct dispatchEntry_s* self)
 {
     UNUSED(self);
 
     if (!ARMING_FLAG(ARMED)) {
         // Don't save if the user made config changes that have not yet been saved.
         if (!isConfigDirty()) {
-            writeEEPROM();
 
-            // Repeat disarming beep indicating the stats save is complete
-            beeper(BEEPER_DISARMING);
+            const bool gyroIsStill = fabsf(gyro.gyroADCf[FD_ROLL]) < systemConfig()->statsSavingMaxGyroRate &&
+                fabsf(gyro.gyroADCf[FD_PITCH]) < systemConfig()->statsSavingMaxGyroRate &&
+                fabsf(gyro.gyroADCf[FD_YAW]) < systemConfig()->statsSavingMaxGyroRate;
+
+            if (gyroIsStill || 0 == systemConfig()->statsSavingMaxGyroRate) {
```

Comment:
What's the difference? Lol

Reply:
@KarateBrot 
I like `CONST == variable` comparison because when you do
`variable == CONST` and accidentally miss =, it will still compile and will give wild results.
If you miss = in `CONST == variable` it won't compile and give you a warning that you trying to assign something to a conts.

Reply:
@limonspb : You get warning for `variable = CONST` for about 25 years in gcc and this style is not used in BF. But the code is correct and it only feels slightly uncommon during review, so IMO it's ok

Reply:
ok i don't wanna break BF coding standards. Changed that.